test/remote/gateways/remote_xpay_test.rb

```diff
@@ -2,14 +2,70 @@
 
 class RemoteRapydTest < Test::Unit::TestCase
   def setup
-    @gateway = XpayGateway.new(fixtures(:x_pay))
-    @amount = 200
-    @credit_card = credit_card('4111111111111111')
-    @options = {}
+    @gateway = XpayGateway.new(fixtures(:xpay))
+    @amount = 100
+    @credit_card = credit_card(
+      '5186151650005008',
+      month: 12,
+      year: 2026,
+      first_name: 'Jim',
+      last_name: 'Smith',
+      verification_value: '123',
+      brand: 'master'
+    )
+
+    @options = {
+      order_id: SecureRandom.alphanumeric(10),
+      billing_address: address,
+      order: {
+        currency: 'EUR',
+        amount: @amount,
+        customer_info: {
+          card_holder_name: 'Jim Smith',
+          card_holder_email: nil
```

Comment:
`email` and `name` are common fields so you shouldn't need these

Reply:
Noted! Changed!